tools/platformio-build.py

```diff
@@ -205,6 +205,14 @@ def scons_patched_match_splitext(path, suffixes=None):
     LIBPATH=[NONOSDK_LIBPATH],
 )
 
+#
+# debug
+#
+for define in env["CCFLAGS"]:
```

Comment:
[definitions are in cppdefines](https://scons.org/doc/latest/HTML/scons-user.html#cv-CPPDEFINES)
could use the flattened ones, or loop env[...] but note of possible types (tuple or string)